### ISSUE TITLE
Table of content links to the old Wiki

### ISSUE THREAD

**Issue by mloskot** [boostorg/website#471]
The Welcome section of the table of content links to the Wiki as follows:

```html
<li><a href="https://svn.boost.org/trac/boost/" class=
    "external">Wiki<span class="link">&gt;</span></a></li>
```

Can we improve it applying one of these?
- Point the link to https://github.com/boostorg/boost/wiki/
- Have "Wiki" and "Old Wiki" separately
- Remove the Wiki link completely

**Comment by pabristow** [comment]
Point the link to https://github.com/boostorg/boost/wiki/

But we need to make sure that the wiki is always up to date ;-)

**Comment by mloskot** [comment]
> But we need to make sure that the wiki is always up to date ;-)

I doubt it is possible, unless GitHub allows to restrict who can edit the Wiki.

For example, someone who I'm unable to identify as an active member of Boost community applied strange changes to the Getting Started wiki page, https://github.com/boostorg/boost/wiki/Getting-Started/_history

![image](https://user-images.githubusercontent.com/80741/74257682-76ad7880-4cf5-11ea-923b-9702a575c7a5.png)

and the page currently displays a rather peculiar content

![image](https://user-images.githubusercontent.com/80741/74257570-51b90580-4cf5-11ea-95c0-5b3260308963.png)

**Comment by pdimov** [comment]
I suggested a separate repository for the wiki at the time, so that we can limit who can edit it, but nobody listened, as usual.

**Comment by mloskot** [comment]
Back to the subject, the ToC link fixed in #472

**Comment by mloskot** [comment]
Following move, yesterday by @glenfe & @pdimov, from https://github.com/boostorg/boost/wiki to https://github.com/boostorg/wiki/wiki, I updated the PR #472
